Dart/src/com/jetbrains/lang/dart/highlight/DartSyntaxHighlighterColors.java

```diff
@@ -54,11 +57,13 @@
   public static final String DART_STATIC_GETTER_REFERENCE = "DART_STATIC_GETTER_REFERENCE";
   public static final String DART_STATIC_METHOD_DECLARATION = "DART_STATIC_METHOD_DECLARATION";
   public static final String DART_STATIC_METHOD_REFERENCE = "DART_STATIC_METHOD_REFERENCE";
+  public static final String DART_STATIC_METHOD_TEAR_OFF = "DART_STATIC_METHOD_TEAR_OFF";
```

Comment:
Should it also be registered in `DartAnnotator`?

Reply:
Added